meson.build

```diff
@@ -0,0 +1,129 @@
+# This file is part of stda.
+#
+# Copyright (C) 2019 Sebastian Ehlert
+#
+# xtb4stda is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# xtb4stda is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with xtb4stda.  If not, see <https://www.gnu.org/licenses/>.
+
+project('stda', 'fortran',
+        version: '1.6.1',
+        license: 'LGPL3',
+        meson_version: '>=0.51')
```

Comment:
if you require version 0.51 here, it should be stated in the README correctly. The real point is you aren't using 0.51 features and this `meson.build` is 0.49 compatible.